satellite/rewards/rewards.go

```diff
@@ -5,26 +5,17 @@ package rewards
 
 import (
 	"context"
-	"fmt"
-	"math"
-	"strconv"
 	"time"
 )
 
 // ToCents converts USD credit amounts to cents.
-func ToCents(dollar int) int {
-	cents := float64(dollar) * 100
-    return int(math.Round(cents))
+func ToCents(dollars int) int {
+    return dollars * 100
 }
 
 // ToDollars converts credit amounts in cents to USD.
 func ToDollars(cents int) int {
-	formattedAmount := fmt.Sprintf("%d.%d",(cents / 100),(cents % 100))
```

Comment:
We still need this format here. Otherwise, `5550` cents would return `55` instead of `55.50`

Reply:
hmm, so then I would be returning either a `float64` or a `string`, neither of which I can re-assign back to  `...CreditInCents` without type-casting and truncating the output.

Reply:
I could do it in the template but then I would definitely have to create the custom type no ?

Reply:
https://github.com/storj/storj/pull/2186#discussion_r298308612